pkg/commands/internal/configuration.go

```diff
@@ -0,0 +1,138 @@
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+const base = ".mygcl"
+
+const defaultBinaryName = "gcl-custom"
```

Comment:
I think the name in the reference yaml is much better default name; `custom-golangci-lint`.

Reply:
I preferred a name consistent with the file configuration name `mygcl` -> `gcl` but if someone uses an alias `gcl` for `golangci-lint it can be confusing.
So I use a "verbose" non-ambiguous name.

Reply:
If you want to keep `mygcl.yml` and `gcl` is to be seen as an abbreviation of `golangci-lint`, wouldn't `mygcl` (or `my-gcl`) be a better name? Otherwise the definition of not upstream for config would be `my` and for binary `custom`. I think using one of them would make sense. 

And seems like this is very related to my first comment about the name of `.mygcl.yml` so let's keep it all in one thread.

We call the feature `custom` so using that instead of `my` would avoid having two references of the custom built tool (my _and_ custom). `.gcl-custom.yml` (or `.custom-gcl.yml`) and generate a binary named `gcl-custom` or keep the name `.mygcl.yml` and generate `mygcl` (or `my-gcl`).

Do you think that's a bad idea?

Reply:
Seems right, `.custom-gcl.yml` for the configuration and `custom-gcl` for the binary name.